sdk/test-utils/recorder/test/node/utils.spec.ts

```diff
@@ -4,12 +4,13 @@ import {
   testHasChanged,
   isContentTypeInNockFixture,
   decodeHexEncodingIfExistsInNockFixture,
-  handleSingleQuotesInUrlPath
+  maskAccessTokenInNockFixture
 } from "../../src/utils";
 
 import { nodeRequireRecordingIfExists, findRecordingsFolderPath } from "../../src/utils/recordings";
 
 import chai, { expect } from "chai";
+import { handleSingleQuotesInUrlPath } from "../../src/utils/index";
```

Comment:
Same comment as Jeremy here about just importing from ../../src/utils

Reply:
I was resolving a bad merge and had this when we had both `utils/index.ts` and `utils.ts`.
Now that I've deleted `utils.ts`, this can be moved.